### PR TITLE
nightly: Add the loongaarch64-unkown-linux-gnu target

### PR DESCRIPTION
**Pull request by tgonzalezorlandoarm**
Fixes nightly issues.
Please refer to https://github.com/parallaxsecond/rust-cryptoki/actions/runs/6923840384

### CODE REVIEW

**Comment by tgonzalezorlandoarm** [comment]
Please see https://github.com/tgonzalezorlandoarm/rust-cryptoki/actions/runs/6928647087 for the internal check on my fork

File: .github/actions/ci_script/action.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    - name: Install Rust targets
+      run: |
+        rustup target add armv7-unknown-linux-gnueabi &&
+        rustup target add armv7-unknown-linux-gnueabihf &&
+        rustup target add arm-unknown-linux-gnueabi &&
+        rustup target add aarch64-unknown-linux-gnu &&
+        rustup target add i686-unknown-linux-gnu &&
+        rustup target add loongarch64-unknown-linux-gnu &&
+        rustup target add powerpc64-unknown-linux-gnu &&
```
[the remark this answers is not in the document]
Comment (tgonzalezorlandoarm) [reply .github/actions/ci_script/action.yml:23]: I don't understand why it's here, but this is not part of the PR, this is part of main

**Comment by tgonzalezorlandoarm** [comment]
https://github.com/tgonzalezorlandoarm/rust-cryptoki/actions/runs/6931514956

**Review by minosgalanakis — APPROVED** [review]
LGTM

**Review by gowthamsk-arm — APPROVED** [review]
LGTM

**Pull request merged**